hypothesis-python/tests/conjecture/test_shrinker.py

```diff
@@ -298,11 +271,8 @@ def shrinker(data):
         if result >= 32768 and cap == 1:
             data.mark_interesting()
 
-    shrinker.fixate_shrink_passes(["minimize_individual_blocks"])
-    # the minimal bitstream here is actually b'\x01\x01\x00\x00\x01\x00\x00\x01',
-    # but the shrinker can't discover that it can shrink the size down from 64
-    # to 32...
-    assert list(shrinker.shrink_target.buffer) == [3, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1]
+    shrinker.fixate_shrink_passes(["minimize_individual_nodes"])
+    assert list(shrinker.shrink_target.buffer) == [1, 1, 0, 0, 1, 0, 0, 1]
```

Comment:
indeed! but don't get too excited, because I think I was the one who wrote that comment, after I had regressed this test at some point in the ir transition...but it's nice to return to the status quo 🙂